pages/learn/manage/actions.md

```diff
@@ -32,14 +32,21 @@ You may also enable or disable public device URLs by clicking the _Public device
 
 The `Restart Application` action restarts the currently running **application containers**. Your application's running containers will be removed and recreated from scratch. This behavior is intended, and is different from running `balena restart [OPTIONS] CONTAINER [CONTAINER...]` in a host OS terminal instance from your dashboard, which will not remove your containers. If you are trying to persist data between container removals, see [persistent storage][persistent-storage] for strategies.
 
-When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM` and after 10 seconds of wait time a `SIGKILL` is sent.
+When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM`. After 10 seconds of wait time, a `SIGKILL` is sent.
 
 __Note:__ During a restart any data that is not stored in `/data` will be lost.
 
 It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
 
 __Warning:__ Restart device container is not equivalent to a reboot of the device!
 
+Here is {{ $names.company.lower }}'s rationale for the restart action being different than Docker's restart action:
```

Comment:
```suggestion
By removing containers and recreating them from scratch we see benefits like the following.
```

Reply:
Can we move this entire block of reasons up to line 34 so it follows the paragraph about it being intended behaviour?

Reply:
Sounds good.

Reply:
I was trying to think of a better way to phrase the `Here is balena's rationale...` line but couldn't think of anything. Thanks!